frontend/src/lib/k8s/cluster.ts

```diff
@@ -15,6 +15,9 @@ export interface Cluster {
   name: string;
   useToken?: boolean;
   [propName: string]: any;
+  notifications?: KubeEvent[];
```

Comment:
Notifications will be for more than KubeEvents. So they should support info related to a KubeEvent but not be limited to it.
i.e. it's better to implement a `Notification` type and have a function that converts a KubeEvent to it.

Reply:
Notifications is also a global feature, meaning each notification **may** be related to a cluster, but we want these to be independent from clusters as we're showing them all together.

Reply:
Fixed